service/src/test/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityUserServiceTest.java

```diff
@@ -40,9 +45,22 @@ void setup() {
 
     @Test
     void shouldGrantCaseRolesToAllLocalAuthorityUsers() {
+        given(featureToggleService.isCaseUserAssignmentEnabled()).willReturn(false);
+
         localAuthorityUserService.grantUserAccessWithCaseRole(CASE_ID, LOCAL_AUTHORITY);
 
         verify(caseRoleService).grantAccessToUser(CASE_ID, USER_ID, CASE_ROLES);
         verify(caseRoleService).grantAccessToLocalAuthority(CASE_ID, LOCAL_AUTHORITY, CASE_ROLES, Set.of(USER_ID));
+        verifyNoMoreInteractions(caseRoleService);
+    }
+
+    @Test
+    void shouldGrantCaseRolesToAllLocalAuthorityUsersWithCaseAssignmentRoles() {
```

Comment:
dont like this method name, it does not tell a lot how it is different from previous one, maybe
shouldBulkGrantCaseRolesToAllLocalAuthorityUsers

Reply:
sorted